app/assets/stylesheets/_event-banner.scss

```diff
@@ -35,9 +35,24 @@
     font-size: $font-size-h3;
     font-weight: lighter;
 
+    a {
+      color: #f89090;
+
+      &:hover {
+        color: #cc342d;
```

Comment:
Color literals like `#cc342d` should only be used in variable declarations; they should be referred to via variable everywhere else.